From b7f32567d8e47a358cf52a95c6883cec458def65 Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Sun, 2 Dec 2018 15:51:54 -0700
Subject: [PATCH 13/13] legacy: Allow platform_app to read
 qemu_hw_mainkeys_prop

* Noted upon launching gallery3d (Gallery2)
* avc: denied { read } for name="u:object_r:qemu_hw_mainkeys_prop:s0"
  dev="tmpfs" ino=7079 scontext=u:r:platform_app:s0:c512,c768
  tcontext=u:object_r:qemu_hw_mainkeys_prop:s0 tclass=file
  permissive=0

Change-Id: I9238cf4061b7b6263a6629584b0581dec34a090b
---
 common/platform_app.te | 1 +
 1 file changed, 1 insertion(+)

diff --git a/common/platform_app.te b/common/platform_app.te
index 2e506da..ed76180 100644
--- a/common/platform_app.te
+++ b/common/platform_app.te
@@ -21,6 +21,7 @@ hal_client_domain(platform_app, hal_hbtp)
 
 #get_prop(platform_app, bluetooth_prop)
 get_prop(platform_app, debug_gralloc_prop)
+get_prop(platform_app, qemu_hw_mainkeys_prop)
 
 #for perf-hal call
 hal_client_domain(platform_app, hal_perf)
-- 
2.17.1

